esmvaltool/recipes/recipe_climwip_test_performance_sigma.yml

```diff
@@ -177,33 +172,24 @@ diagnostics:
         script: weighting/weighted_temperature_graph.py
         ancestors: [calculate_weights_climwip/climwip, tas]
         weights: 'weights.nc'
+        settings:
+          <<: *ref_period
+          central_estimate: mean
+          lower_bound: 16.7
+          upper_bound: 83.3
 
   weighted_temperature_map:
     description: >
       Plot map of weighted multi-model mean and difference between weighted
       and unweighted multi-model mean.
-    themes:
-      - phys
-    realms:
-      - atmos
-
-    variables:
-      tas: &map_settings
-        short_name: tas
-        start_year: 2081
-        end_year: 2100
-        mip: Amon
-        preprocessor: climatological_mean
-      tas_reference:
-        <<: *map_settings
-        start_year: 1995
-        end_year: 2014
     scripts:
       weighted_temperature_map:
         script: weighting/weighted_temperature_map.py
-        ancestors: [calculate_weights_climwip/climwip, tas, tas_reference]
+        ancestors: [calculate_weights_climwip/climwip,
+                    calculate_weights_climwip/tas_CLIM_future,
+                    calculate_weights_climwip/tas_CLIM_reference]
```

Comment:
If this diagnostic only runs a script, why not add that script to the other diagnostic (the one that makes all its ancestors)?

Reply:
I was trying to keep the different ClimWIP recipes as consistent as possible. The case where all variables already exist is a special case (e.g. in recipe_climwip_test_basic.yml this is not the case). But if you want I can do that - although then   weighted_temperature_map would be a script in calculate_weights_climwip which does not seem too intuitive?

Reply:
Okay this works for me, especially considering the consistent structure of the ESMValTool output directory 👍